src/test/Pricing.java

```diff
@@ -2,61 +2,61 @@
 
 import java.time.DayOfWeek;
 import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
 
 import static java.lang.Math.round;
 
 public class Pricing {
-    private static final String sexM = "男";
-    private static final String sexF = "女";
-//----------必要に応じて条件を変更する際はSetの値を変更する。------------------//
-    private static final  String sexSet = sexM;
+    private static final String SEX_M = "男";
+    private static final String SEX_F = "女";
+//-----------------------必要に応じて条件を変更。-----------------------------//
+    private static final  String SEX_SETTING = SEX_M;
 
-    private static final int ageSet = 13;
-    private static final String eventDaySet = "木曜日";
+    private static final int AGE_SETTING = 13;
+    private static final String EVENT_DAY = "土曜日";
 
-    private static final int basePriceSet = 2000;
+    private static final int BASE_PRICE_SET = 2000;
 //----------------------------------------------------------------------//
-    private static String DayJpn() {
-        LocalDate now = LocalDate.now();
-        DayOfWeek day = now.getDayOfWeek();
-        String dayJpn = "";
-        if (day == DayOfWeek.MONDAY) {
-            dayJpn = "月曜日";
-        } else if (day == DayOfWeek.TUESDAY) {
-            dayJpn = "火曜日";
-        } else if (day == DayOfWeek.WEDNESDAY) {
-            dayJpn = "水曜日";
-        } else if (day == DayOfWeek.THURSDAY) {
-            dayJpn = "木曜日";
-        } else if (day == DayOfWeek.FRIDAY) {
-            dayJpn = "金曜日";
-        } else if (day == DayOfWeek.SATURDAY) {
-            dayJpn = "土曜日";
-        } else if (day == DayOfWeek.SUNDAY) {
-            dayJpn = "日曜日";
-        } return dayJpn;
-    }
+    private static String CallDayWeek() {
+         LocalDate localDate = LocalDate.now();
+         DateTimeFormatter formmater = DateTimeFormatter.ofPattern("EEEE", Locale.JAPANESE);
+         String format = localDate.format(formmater);
+    return format;
+}
+    public static void info(CustomerForm Classify) {
```

Comment:
変数名は小文字から始めます。
また変数名は原則名詞が望ましいですが、classifyは分類すると言う動詞です。
customerFormで良いかと思います。
```suggestion
    public static void info(CustomerForm classify) {
```